src/app/data-planes/views/DataPlaneInboundSummaryView.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <RouteView
+    v-slot="{ route }"
+    name="data-plane-inbound-summary-view"
+    :params="{
+      service: '',
+    }"
+  >
+    <AppView>
+      <template #title>
+        <div class="summary-title-wrapper">
+          <img
+            aria-hidden="true"
+            src="@/assets/images/icon-wifi-tethering.svg?url"
+          >
+
+          <h2 class="summary-title">
+            Inbound :{{ route.params.service }}
```

Comment:
**Question**: I noticed this in a few PRs recently: is it intentional that you’re not putting UI texts into the message files?

Reply:
Oh sorry, no its just that this is mainly still in prototyping mode. So I'm not thinking about i18n etc. until we are sure we know what we are showing, otherwise we'll accidentally end up with loads of unused i18n texts (we have accumulated a few of those already I think). I think we are just about coming out of prototype mode, so I'll probably have PR that fills all these in pretty soon.

Oh you've just reminded my of another question I had, Its unrelated to this though so I'll make a new comment.